src/widgets/hierarchical-menu/hierarchical-menu.tsx

```diff
@@ -335,6 +339,10 @@ const hierarchicalMenu: HierarchicalMenuWidget = function hierarchicalMenu(
       userCssClasses.disabledShowMore
     ),
   };
+  const templates = {
+    ...defaultTemplates,
+    ...userTemplates,
+  };
```

Comment:
not needed

Reply:
https://github.com/algolia/instantsearch.js/pull/4785/commits/61ea462ec076c7fd7a38a33f8d0bde9344d72d0a